deployments/liqo/templates/auth-service-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: auth-service
+    app: liqo.io
+  name: auth-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: auth-service
+  template:
+    metadata:
+      labels:
+        run: auth-service
+    spec:
+      serviceAccountName: auth-service-sa
+      {{ if not .Values.authService.ingress.enable }}
+      initContainers:
+        - name: cert-creator
+          image: nginx:1.19
+          volumeMounts:
+            - mountPath: '/certs'
+              name: certs
+          command: [ "/bin/sh" ]
+          args: [ "-c", 'openssl req -x509 -subj "/C=IT/ST=Turin/O=Liqo" -nodes -days 365 -newkey rsa:4096 -keyout /certs/key.pem -out /certs/cert.pem' ]
+      {{ end }}
+      containers:
+        - image: {{ .Values.authService.image.repository }}{{ .Values.global.suffix | default .Values.suffix }}:{{ .Values.global.version | default .Values.version }}
+          name: auth-service
+          imagePullPolicy: {{ .Values.pullPolicy }}
+          command: ["/usr/bin/auth-service"]
+          args:
+          - "--namespace"
+          - "$(POD_NAMESPACE)"
+          - "--resyncSeconds"
+          - "29"
+          - "--listeningPort"
+          {{ if .Values.authService.ingress.enable }}
+          - "4999"
+          {{ else }}
+          - "442"
+          - "--useTls"
```

Comment:
Is there any reason why these 3 numbers have been decreased by 1? (the same for the ingress and the service)

Reply:
very strange